default.nix

```diff
@@ -53,17 +57,9 @@ let
     cardano-sl-explorer = prodMode (super.callPackage ./pkgs/cardano-sl-explorer.nix { });
     cardano-sl-explorer-static = justStaticExecutables self.cardano-sl-explorer;
 
-    #mkDerivation = args: super.mkDerivation (args // {
-    #enableLibraryProfiling = false;
-    #});
-  };
-});
-  iohk-ops-extra-runtime-deps = [
-    pkgs.git pkgs.nix-prefetch-scripts
-  ];
-  cabal2nixpkgs = rec {
-    # extra packages to expose, that have no relation to pkgs/default.nix
-    stack2nix = compiler.callPackage ./pkgs/stack2nix.nix {};
+    iohk-ops-extra-runtime-deps = [
+      pkgs.git pkgs.nix-prefetch-scripts self.yaml
+    ];
     iohk-ops = pkgs.haskell.lib.overrideCabal
```

Comment:
with this change, `iohk-ops` now depends on the versions cardano's stack.yaml picked, and updating `pkgs/default.nix` will invalidate the cached build of `iohk-ops`